vhdbuilder/packer/install-dependencies.sh

```diff
@@ -200,13 +200,10 @@ for AZURE_CNI_NETWORKMONITOR_VERSION in ${AZURE_CNI_NETWORKMONITOR_VERSIONS}; do
     echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
 done
 
-AZURE_NPM_VERSIONS="
-1.1.2
-1.1.0
-1.0.33
-1.0.32
-1.0.30
-"
+#Get the last 5 versions from azure-npm
+#This scipt sure isn't pretty so open to sugestestiosn to make it more elegant
+AZURE_NPM_VERSIONS=$(curl -s -L  https://mcr.microsoft.com/v2/containernetworking/azure-npm/tags/list | jq --raw-output '.tags | map(select(contains("-")!=true))[-5:] | .[] ')
```

Comment:
If you really want to get some automatic caching, maybe add this on top of the existing static list and adjust the static list periodically but much less frequently